tests/mail/tests.py

```diff
@@ -393,6 +409,23 @@ def test_to_in_headers_only(self):
         ).message()
         self.assertEqual(message["To"], "to@example.com")
 
+    def test_to_not_repeated(self):
+        email = EmailMessage(
+            subject="test subject",
+            body="test body",
+            from_email="from@example.com",
+            to=["guy1@example.com"],
+            headers={"To": "guy1@example.com, guy2@example.com, guy3@example.com"},
+        )
+        message = email.message()
+        count_to = len(message.get_all("To"))
+
+        self.assertEqual(count_to, 1)
```

Comment:
I would assert values not only the number of headers
```suggestion
        self.assertEqual(
            message.get_all("To"),
            ["guy1@example.com, guy2@example.com, guy3@example.com"],
        )     
```